sweepbatcher/sweep_batch.go

```diff
@@ -137,6 +137,12 @@ type batchConfig struct {
 	// the caller has to update it in the source of SweepInfo (interface
 	// SweepFetcher) and re-add the sweep by calling AddSweep.
 	noBumping bool
+
+	// signMuSig2 is a custom signer. If it is set, it is used to create
+	// musig2 signatures instead of musig2SignSweep and signerClient. Note
+	// that musig2SignSweep must be nil in this case, however signerClient
+	// must still be provided, as it is used for non-coop spendings.
+	signMuSig2 SignMuSig2
```

Comment:
can we rename this to `customMuSig2Signer`? Lines like `if cfg.signMuSig2 != nil && musig2ServerSigner != nil` then become easier to read.

Reply:
Fixed